scripts/bump-version.py

```diff
@@ -23,8 +23,9 @@
 
 PYOLITE_PACKAGE = ROOT / "packages" / "pyolite-kernel"
 PYOLITE_PACKAGE_JSON = PYOLITE_PACKAGE / "package.json"
-PYOLITE_KERNEL_SOURCE = PYOLITE_PACKAGE / "src" / "kernel.ts"
+PYPI_SOURCE = PYOLITE_PACKAGE / "src" / "_pypi.ts"
```

Comment:
hmmm... this file gets generated.

Reply:
The real source of truth migh tbe: https://github.com/jupyterlite/jupyterlite/blob/main/packages/pyolite-kernel/package.json#L69 (unless that's covered above)

Reply:
Maybe this one is not needed actually, gonna double check.

Reply:
This gets regenerated by `doit build:js:piplite:wheels`, so if that happens after this runs...